rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
```

Comment:
"Model social norms" means that team members are expected to make sure that their behavior reflects the statement of values and sets a good example for everyone.

Reply:
besides the removal of "model social norms" i think this is a reasonable suggestion, since we already talk about prioritizing mediation over bans or other moderation actions elsewhere in the document.

Reply:
So «model» could be replaced with «serve as an example of social norms»? I also find «model» a confusing word.

Reply:
I believe we've addressed both parts of this as of the latest changes, please take a look.